### DISCUSSION TITLE
Hosting a site

### DISCUSSION THREAD

**Discussion by dumgyan** [community/community#28981] [Other Feature Feedback, Questions, & Ideas]
Pls how long would it take for my site to be published? Is 6 hours now, still I can't access it.

**Comment by radar-blip** [comment]
Which one is it? Are you using Github Pages?

**Reply by radar-blip** [reply]
It seems you have got it working now. 👍🏼

**Comment by jfullstackdev** [comment]
it's already being hosted 

![image](https://user-images.githubusercontent.com/47092464/184071292-7f8519a2-1771-4afa-b202-cc72e595dbce.png)

the problem is that, it's not the same repo name as your username, so it will be at the root of your GitHub Pages,

rename the repo from `rabbitman.github.io` to `rabbikojodumgyan.github.io`
